### DISCUSSION TITLE
Should overlay close when its ancestor scrolls?

### DISCUSSION THREAD

**Discussion by hunterloftis** [adobe/spectrum-web-components#2889] [General]
We decided *yes* not too long ago: https://github.com/adobe/spectrum-web-components/pull/2736

But since then we've seen at least one use-case where an overlay was wanted to stay open during scrolling.

/cc @Westbrook @najikahalsema

**Comment by pwong09** [comment]
It would be great to keep the overlay open, but its sizing should be static when user scrolls outside of the overlay - maybe add an attribute for height size (s, m, l)?
